app/src/main/java/de/xikolo/models/dao/UserDao.kt

```diff
@@ -1,13 +1,21 @@
 package de.xikolo.models.dao
 
+import androidx.lifecycle.LiveData
 import de.xikolo.extensions.asCopy
+import de.xikolo.extensions.asLiveData
+import de.xikolo.managers.UserManager
 import de.xikolo.models.User
 import de.xikolo.models.dao.base.BaseDao
 import io.realm.Realm
 import io.realm.kotlin.where
 
 class UserDao(realm: Realm) : BaseDao<User>(User::class, realm) {
 
+    fun getCurrent(): LiveData<User> = query()
```

Comment:
Either `findCurrent` or simply `current`.

Reply:
Renamed to `current`